test/k8s/egress.go

```diff
@@ -76,8 +77,23 @@ var _ = SkipDescribeIf(func() bool {
 		res.ExpectSuccess()
 	}
 
-	BeforeAll(func() {
+	// ctEntriesOnNode returns the number of CT entries matching destination
+	// IP and port on a given node
+	ctEntriesOnNode := func(node, dstIP, dstPort string) int {
+		ciliumPod, err := kubectl.GetCiliumPodOnNode(node)
+		Expect(err).Should(BeNil(), "Unable to determine cilium pod on node %s", node)
+
+		cmd := fmt.Sprintf("cilium bpf ct list global | grep '\\-> %s:%s' | wc -l", dstIP, dstPort)
```

Comment:
Worth making sure the port isn't matching on more than intended?
```
grep '\\-> %s:%s\\b'
```
We've had such bugs in the past with IP addresses and it caused a long-lived flake.

Reply:
Right, nice catch :boom: